PowerEditor/src/MISC/PluginsManager/PluginsManager.cpp

```diff
@@ -78,11 +78,11 @@ int PluginsManager::loadPlugin(const TCHAR *pluginFilePath, vector<generic_strin
 	    pi->_hLib = ::LoadLibrary(pluginFilePath);
         if (!pi->_hLib)
         {
-            const std::wstring& lastErrorMsg = GetLastErrorAsString();
+            generic_string lastErrorMsg = GetLastErrorAsString();
             if (lastErrorMsg.empty())
                 throw generic_string(TEXT("Load Library is failed.\nMake \"Runtime Library\" setting of this project as \"Multi-threaded(/MT)\" may cure this problem."));

```

Comment:
Sorry, didn't notice this message. Let me correct this as well.
This PR is growing like anything.

Anyway, I'm using web for pull and push. Is there any good tool to manage (apart from git Gui and Git Desktop)? Or is there any way to get GitDesktop offline installer as online install helper is not working for me? Please don't mind, I'm novice to github.